hack/golang/test.sh

```diff
@@ -2,24 +2,23 @@
 
 set -e
 
-CGO_ENABLED=1
-
 perform_tests() {
-  echo "Performing tests"
-  go test -v -covermode=atomic -coverprofile=coverage.txt ./...
+  echo "Performing tests on $1"
+  GODEBUG=gocachetest=1 go test -v -covermode=atomic -coverprofile=coverage.txt "$1"
```

Comment:
oops this is not intentional (`GODEBUG`), I was trying to debug cache, and left it by mistake